### PR TITLE
added appleWatchFaces

### PR DESCRIPTION
**Pull request by orff**
1. [x] Project URL:
2. [x] Update contents.json instead of README 
3. [x] One project per pull request
4. [x] Screenshot included 
5. [x] Avoid iOS or open-source in description as it is assumed
6. [x] Use this commit title format if applicable: Add app-name by @github-username
7. [x] Use approved format for your entry

### CODE REVIEW

**Comment by orff** [comment]
hope I got it

File: contents.json [hunk 1]
```diff
@@ -17200,6 +17200,25 @@
...
       "date_added": "Dec 10 2018",
       "suggested_by": "@marty-suzuki"
-     }
+     },
+     {
+      "title": "AppleWatchFaces",
+      "category-ids": [
+        "apple-watch"
+      ],
+      "description": "Apple Watch Face Editor iOS and watchOS apps",
```
Comment (vpeschenkov) [inline contents.json]: Hi @orff, could you use another description text? This one doesn't explain what the app does.

**Comment by vpeschenkov** [comment]
Hi @orff, could you fix the review issue and resolve merge conflicts?

File: contents.json (hunk 1 above)
Comment (orff) [reply contents.json]: Will do

**Comment by orff** [comment]
updated with better description

**Pull request closed without merging**